src/precompiles/ArbWasmCache.sol

```diff
@@ -16,11 +16,14 @@ interface ArbWasmCache {
     /// @return managers the list of managers.
     function allCacheManagers() external view returns (address[] memory managers);
 
-    /// @notice Caches all programs with the given codehash.
+    /// @dev Deprecated, replaced with cacheProgram
+    function cacheCodehash(bytes32 codehash) external;
```

Comment:
What about completely removing this now?  Only on testnet, so not the end of the world if some things break until updated

Reply:
We want to maintain the ability to sync past the testnet's ArbOS 30 section (for now at least)